test/decl/protocol/special/coding/Inputs/class_codable_simple_multi1.swift

```diff
@@ -0,0 +1,25 @@
+// REQUIRES: objc_interop
```

Comment:
Yes, I believe so. swift-corelibs-foundation will require some changes to bring it in line with the implementation of `JSONEncoder` and `JSONDecoder` specifically, but the primary issue in testing is not one of actual ObjC interop, but one of the testing framework.

All of my implementations should be able to land in swift-corelibs-foundation as well with no changes.